src/components/NewNursery/index.tsx

```diff
@@ -162,18 +138,10 @@ export default function NurseryView(): JSX.Element {
             </Grid>
             {timeZoneFeatureEnabled && (
               <Grid item xs={gridSize()}>
-                <TimeZoneSelector
-                  selectedTimeZone={record?.timeZone || defaultTimeZone.id}
-                  onTimeZoneSelected={onChangeTimeZone}
-                  label={strings.TIME_ZONE}
-                  disabled={orgTZChecked}
-                />
-                <Checkbox
-                  label={strings.USE_ORG_TZ}
-                  onChange={(value) => onOrgTimeZoneChecked(value)}
-                  id='orgTZ'
-                  name='orgTZ'
-                  value={orgTZChecked}
+                <TimeZoneSelectorWithCheckbox
+                  record={record}
+                  setRecord={setRecord}
```

Comment:
i know we have been using record/setRecord as props in many places.
These are semantics local to the component.
Ideally the props should be in the time zone selector component's semantics.
example:
```
  location={record}
```
we don't need to expose setState semantics etc.
remove setRecord?